dev/com.ibm.ws.security.utility/resources/com/ibm/ws/security/utility/resources/UtilityMessages.nlsprops

```diff
@@ -86,6 +86,16 @@ encode.option-desc.custom.description=\
 # encryption is enabled.
 no.custom.encyption=The custom password encryption is not detected.
 
+# The following messages are for retrieving an encryption key from a 
```

Comment:
for all messages in this set, only need one space after a period.

Reply:
I've updated the double spaces in that section,  please review.